components/team/ExecCard.jsx

```diff
@@ -3,27 +3,24 @@ import Image from "next/image";
 
 import styles from "styles/modules/ExecCard.module.scss";
 
-// TODO: consider separating into component that's more generic, and having
-// ExecCard inherit from that
+// TODO: consider separating into component that's more generic, and having ExecCard inherit from that
 const ExecCard = ({ data, colour }) => {
   const { name, image, description, role } = data;
 
-  // TODO: fix description not showing
   return (
     <div className="col-md-4">
       <div className="card-person">
         <div className={`${styles["card-person-style" + colour]} limit`}>
-          <div className={styles["card-person-animation"]} />
-          <Image alt={name} src={image} className="card-img-top" />
-          <div className={styles[`go-corner-style${colour}`]}>
-            <div className={styles["go-arrow"]} />
-          </div>
+          {description === "" || description === undefined ? null : (
+            <div className={styles["card-person-animation"]} />
+          )}
```

Comment:
Could be worth looking into this
```suggestion
          { description && <div className={styles["card-person-animation"]} /> }
```

Reply:
Corrected before review went in :P

(thanks)

Reply:
Bro's got that -10000ms response time